app/assets/stylesheets/custom.scss.erb

```diff
@@ -5044,3 +5044,35 @@ input:required:valid {
   font-weight: bold;
   text-transform: uppercase;
 }
+
+.enr-pvp-indicator {
+  width: 30%;
+  border: 2px solid #247dc5;
+  background-color: #247dc51a;
+  border-radius: 0.5em;
+  padding: 0.3em;
+  color: #247dc5;
+  font-size: 12px;
+  text-align: center;
+  font-weight: bold;
+  text-transform: uppercase;
+  margin-left: 1.2em;
+  margin-bottom: 0.5em;
+  margin-top: -0.7em
+}
+
+.enr-standard-plan-indicator {
```

Comment:
There's so much the same with these two classes (though not width, which surprised me - this may relate to my confusion below on bookmark vs square icons), makes me wonder if a separate class for the common aspects (or Sass placeholder+@extend) makes sense. Doesn't meet the Rule of Three or anything - feel free to disregard unless the idea moves you.